### PR TITLE
Add FunctorMaybe instance to Maybe

### PR DESCRIPTION
**Pull request by alexfmpe**

### CODE REVIEW

File: src/Reflex/FunctorMaybe.hs [hunk 1]
```diff
@@ -24,6 +24,10 @@ class FunctorMaybe f where
   -- | Combined mapping and filtering function.
   fmapMaybe :: (a -> Maybe b) -> f a -> f b
 
+-- | @fmapMaybe = flip (>>=)
+instance FunctorMaybe Maybe where
+  fmapMaybe = flip (>>=)
```
Comment (3noch) [inline src/Reflex/FunctorMaybe.hs]: `(=<<)`?

Comment (ryantrinkle) [reply src/Reflex/FunctorMaybe.hs]: Good idea :)

Comment (alexfmpe) [reply src/Reflex/FunctorMaybe.hs]: Derp. Fixed.

**Comment by ryantrinkle** [comment]
Thanks!

**Pull request merged**